### PR TITLE
Revert "split sync data e2e to avoid timeout (#4458)"

### PR DESCRIPTION
**Pull request by cmonfortep**
Task/Issue URL: https://app.asana.com/0/488551667048375/1207168620924846/f 

### Description
Revert changes

### Steps to test this PR

_Feature 1_
- [ ]
- [ ]

### UI changes
| Before  | After |
| ------ | ----- |
!(Upload before screenshot)|(Upload after screenshot)|

### CODE REVIEW

**Comment by cmonfortep** [comment]
* **#4465** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4465?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @cmonfortep and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .github/workflows/sync-critical-path.yml [hunk 1]
```diff
@@ -72,13 +72,6 @@ jobs:
           env: |
             CODE=${{ steps.sync-recovery-code.outputs.recovery-code }}
 
-      - name: Maestro Output
```
Comment (malmstein) [inline .github/workflows/sync-critical-path.yml:75]: you don’t need to remove this, it’s actually useful.

**Review by malmstein — APPROVED** [review]
don’t remove the outputs, rest is ok.

File: .github/workflows/sync-critical-path.yml (hunk 1 above)
Comment (cmonfortep) [reply .github/workflows/sync-critical-path.yml:75]: Agree, but I prefer a clean revert for now, and I can bring them back on a new PR. I

**Pull request merged**